metaseq/tasks/streaming_finetune_language_modeling.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
```

Comment:
let's please leave an issue saying to have someone possibly refactor this in the future

Reply:
Will add an issue once this is landed